frontend/packages/kubevirt-plugin/integration-tests-cypress/view/actions.ts

```diff
@@ -6,10 +6,10 @@ export const confirmButton = '[data-test="confirm-action"]';
 export const confirmCloneButton = 'button[id="confirm-action"]';
 export const kebabButton = 'kebab-button';
 
-export const selectItemFromDropdown = (action: string, selector: string) => {
+export const selectItemFromDropdown = (item: string, selector: string) => {
```

Comment:
nitpick: I'm not sure if the `TestActionID` is used on all dropdowns. If it isn't, then this function is actions-specific and I think we should use names to indicate that, such as `selectActionFromDropdown` and leave `item` as `action`. If `TestActionID` is used on all dropdowns, then I think we should move this function to a more common location or use an existing function if one is available.

Reply:
I agree with you to change the function name to `selectActionFromDropdown` and use the parameter `action`.
For now, let it sits here until we have a clear requirement to put it into `utils/utils.ts`.